### ISSUE TITLE
After clicking with the mouse, the editable area becomes uneditable

### ISSUE THREAD

**Issue by yu2chuan** [binderus/binderus#10]
On Windows, I add a new file and click the editable area by mouse.
Then the editable area become un-editable.
I need to choose the file again on the left panel. Then, the file become editable again.

**Comment by binderus** [comment]
Hi @yu2chuan Thanks for reporting this.
This should be fixed in the latest version: https://binderus.com

**Comment by yu2chuan** [comment]
it's so great to receive your response. will you continue this project? I think the UI is so nice.

**Comment by binderus** [comment]
Yes, definitely! There was some delay due to other work, but all is good now.
Expect more useful features in the coming weeks.
Thanks for your support. Cheers.
